src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -208,6 +208,10 @@ def __call__(self, poller):
         correlation_message = ''
         self.progress_controller.begin()
         correlation_id = None
+
+        base = azlogging.get_az_logger()
```

Comment:
It's the az_logger parent, because the children loggers don't get references to the handlers added.